examples/nextjs-live-cursors-advanced/components/LiveCursors.tsx

```diff
@@ -83,21 +86,21 @@ export default function LiveCursors({ cursorPanel }: Props) {
         /**
          * Iterate over other users and display a cursor based on their presence
          */
-        others.map(({ connectionId, presence, info }) => {
-          if (presence == null || presence.cursor == null) {
+        others.map(([id, other]) => {
+          if (other.cursor == null) {
             return null;
           }
 
           return (
             <Cursor
               variant="name"
-              name={info?.name}
-              key={`cursor-${connectionId}`}
+              name={other.info.name}
+              key={`cursor-${id}`}
```

Comment:
You don't even need to prefix this one. This might be copy/pasted originally from another place where this was necessary.

```suggestion
              key={id}
```